src/backend/dx12/src/command.rs

```diff
@@ -368,25 +357,50 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
         }
     }
 
-    fn clear_color(
+    fn clear_color_image(
         &mut self,
-        rtv: &n::RenderTargetView,
+        image: &n::Image,
         _: image::ImageLayout,
-        color: ClearColor,
+        range: image::SubresourceRange,
+        value: com::ClearColor,
     ) {
-        self.clear_render_target_view(rtv.handle, color, &[]);
+        assert!((image::ASPECT_COLOR).contains(range.aspects));
+        assert_eq!(range, image.to_subresource_range(range.aspects));
+        if range.aspects.contains(image::ASPECT_COLOR) {
```

Comment:
No need for this `if` clause we already check for `ASPECT_COLOR` above.

Reply:
the check is non-trivial if the aspect mask is empty

Reply:
(As above)